functions-and-operators/string-functions.md

```diff
@@ -163,11 +163,66 @@ Insert a substring at the specified position up to the specified number of chara
 
 ### [`INSTR()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_instr)
 
-Return the index of the first occurrence of substring.
+The `INSTR(str, substr)` function is used to get the index of the first occurrence of the second given argument `substr` 
+in the first given argument `str`.
+Each argument can be either a string or a number.
+This function is the same as the two-argument version of [`LOCATE(substr, str)`](https://docs.pingcap.com/tidb/dev/string-functions#locate), but with the order of the arguments reversed.
+
+> **Note:**
+>
+> `INSTR(str, substr)` is case sensitive since TiDB uses binary collations, which differs from MySQL.
```

Comment:
```suggestion
> `INSTR(str, substr)` is case sensitive since [TiDB uses binary collations by default](/character-set-and-collation.md), which differs from MySQL.
```

Reply:
```
mysql> SELECT TIDB_VERSION()\G
*************************** 1. row ***************************
TIDB_VERSION(): Release Version: v7.5.0
Edition: Community
Git Commit Hash: 069631e2ecfedc000ffb92c67207bea81380f020
Git Branch: heads/refs/tags/v7.5.0
UTC Build Time: 2023-11-24 08:50:14
GoVersion: go1.21.3
Race Enabled: false
Check Table Before Drop: false
Store: tikv
1 row in set (0.00 sec)

mysql> SHOW VARIABLES LIKE 'collation\_%';
+----------------------+--------------------+
| Variable_name        | Value              |
+----------------------+--------------------+
| collation_connection | utf8mb4_0900_ai_ci |
| collation_database   | utf8mb4_bin        |
| collation_server     | utf8mb4_bin        |
+----------------------+--------------------+
3 rows in set (0.00 sec)

mysql> SELECT INSTR("apple pear apricot", "EAR");
+------------------------------------+
| INSTR("apple pear apricot", "EAR") |
+------------------------------------+
|                                  8 |
+------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT INSTR("apple pear apricot" COLLATE utf8mb4_bin, "EAR");
+--------------------------------------------------------+
| INSTR("apple pear apricot" COLLATE utf8mb4_bin, "EAR") |
+--------------------------------------------------------+
|                                                      0 |
+--------------------------------------------------------+
1 row in set (0.00 sec)
```